cartridges/bm_algolia/cartridge/templates/resources/algolia.properties

```diff
@@ -48,6 +53,11 @@ algolia.label.preference.enablePricingLazyLoad.help=When enabled, prices are fet
 algolia.label.preference.indexoutofstock.help=Index out of stock products. When disabled, only in-stock (ATS higher than the InStock Threshold) products are indexed.
 algolia.label.preference.enableRealTimeInventoryHook.help=Sends inventory updates to Algolia in real-time after each order.
 
+# Algolia_RecordModel options display names
+algolia.preference.recordmodel.variantlevel=Variant-level (one record per variant)
+algolia.preference.recordmodel.masterlevel=Master-level (one record per master)
```

Comment:
Why would you want to change this naming? First it's unrelated to what we've discussed, but most importantly it's not a wording that is used in SFCC UI anymore, they refer to "Base Product" in most places.